src/main/java/frc/robot/commands/GatherTheCube.java

```diff
@@ -4,21 +4,31 @@
 
 package frc.robot.commands;
 
+import frc.robot.subsystems.EndEffector;
+import frc.robot.subsystems.Arm;
+import frc.robot.Constants;
+
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
 public class GatherTheCube extends CommandBase {
+  EndEffector cubeHarvester = new EndEffector();
+  private Arm robotArm = new Arm();
```

Comment:
shouldn't subsystems be initialized in RobotContainer and passed into the command through the constructor arguments?

Reply:
This will help keep us from accidentally having two instances of Arm or EndEffector which could confuse the system.